Sources/CustomDump/Conformances/CoreImage.swift

```diff
@@ -0,0 +1,132 @@
+        @unknown default:
+            return "CIQRCodeDescriptor.ErrorCorrectionLevel.(@unknown default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
+
+extension CIDataMatrixCodeDescriptor.ECCVersion: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        
+        case .v000:
+            return "CIDataMatrixCodeDescriptor.ECCVersion.v000"
+        case .v050:
+            return "CIDataMatrixCodeDescriptor.ECCVersion.v050"
+        case .v080:
+            return "CIDataMatrixCodeDescriptor.ECCVersion.v080"
+        case .v100:
+            return "CIDataMatrixCodeDescriptor.ECCVersion.v100"
+        case .v140:
+            return "CIDataMatrixCodeDescriptor.ECCVersion.v140"
+        case .v200:
+            return "CIDataMatrixCodeDescriptor.ECCVersion.v200"
+        @unknown default:
+            return "CIDataMatrixCodeDescriptor.ECCVersion.(@unknown default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
+extension CIContextOption: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIContextOption.(default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
+extension CIImageRepresentationOption: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIImageRepresentationOption.(default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
+extension CIFormat: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIFormat.(default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
+extension CIImageOption: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIImageOption.(default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
+extension CIImageAutoAdjustmentOption: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIImageAutoAdjustmentOption.(default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
+extension CIKernel: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIKernel.(default, rawValue: \(self))"
+        }
+    }
+}
+
+extension CGRect: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CGRect.(default, rawValue: \(self))"
+        }
+    }
+}
+
+extension CIRAWFilterOption: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIRAWFilterOption.(default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
```

Comment:
This, too, I think can lean on the default rendering behavior, right? It's also a type that's been deprecated.
```suggestion
```

Reply:
That's ok!